### ISSUE TITLE
problems with letsencrypt

### ISSUE THREAD

**Issue by airdogvan** [jitsi/docker-jitsi-meet#684]
I know this is my problem and not the setup in the files on the repository.

But for the life of me can't figure out what's wrong.

The web_1 log gives me this: 

Challenge failed for domain xx.mydomain.example
http-01 challenge for xx.mydomain.com
Cleaning up challenges
Some challenges have failed.

IMPORTANT NOTES:
 - The following errors were reported by the server:

   Domain: xx.mydomain.com
   Type:   connection
   Detail: Fetching
   http://xx.mydomain.com/.well-known/acme-challenge/30lwmuJ2A8VcZDgj7nVhQhDrXLDh27oXv0wg-S8bRwI:
   **Connection refused**

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A/AAAA record(s) for that domain
   contain(s) the right IP address. and much more about firewall etc

My DNS entries are all correct, checked that and my local config files a thousand times, no firewall yet (new system firewall will be up later).

Below the pertinent portion of my .env:

 Basic configuration options

Directory where all configuration will be stored
CONFIG=~/.jitsi-meet-cfg

 Exposed HTTP port
HTTP_PORT=8000

Exposed HTTPS port
HTTPS_PORT=8443

System time zone
TZ=UTC

Public URL for the web service
PUBLIC_URL=https://xx.mydomain.com

 IP address of the Docker host
 See the "Running behind NAT or on a LAN environment" section in the README
DOCKER_HOST_ADDRESS=xxx.xxx.xxx.xx

 Let's Encrypt configuration

 Enable Let's Encrypt certificate generation
ENABLE_LETSENCRYPT=1

 Domain for which to generate the certificate
LETSENCRYPT_DOMAIN=xx.mydomain.com

 E-Mail for receiving important account notifications (mandatory)
LETSENCRYPT_EMAIL=email@xx.com

Haven't touched any other files in the rep, just git cloned.

Any help greatly appreciated, can't figure out why "connection refused".  Done many many certificates before and never had this particular problem.  
Is it normal that when trying the challenge it's connecting to http instead of https ?

Anyway any clues are welcomed.

Thanks
Had to get rid of the comment markers as they were interpreted as bold...

**Comment by airdogvan** [comment]
I noticed in release 4416 the addition of certbot-auto.  Don't know about the dates but I know I used that same rep about 6 months ago and didn't have any problems and am wondering if this addition might change things for me.
I also read another issue which was mentioning problems with Cloudflare not accepting root challenges and I happen to be with them so wondering if that would be my issue.

If this is the case any input on how to solve it (short of changing NS servers) would help.
